src/bigquery.ts

```diff
@@ -640,7 +640,8 @@ export class BigQuery extends Service {
           break;
         }
         case 'TIMESTAMP': {
-          value = BigQuery.timestamp(value);
+          const pd = new PreciseDate(BigInt(value) * BigInt(1000));
+          value = BigQuery.timestamp(pd);
```

Comment:
This is similar on how it was working before this PR: https://github.com/googleapis/nodejs-bigquery/pull/1192/files